selfdrive/ui/sunnypilot/quietmode.py

```diff
@@ -0,0 +1,39 @@
+"""
+Copyright (c) 2021-, Haibin Wen, sunnypilot, and a number of other contributors.
+
+This file is part of sunnypilot and is licensed under the MIT License.
+See the LICENSE.md file in the root directory for more details.
+"""
+from cereal import car
+
+from openpilot.common.params import Params
+
+AudibleAlert = car.CarControl.HUDControl.AudibleAlert
+
+ALERTS_ALWAYS_PLAY = {
+  AudibleAlert.warningSoft,
+  AudibleAlert.warningImmediate,
+  AudibleAlert.promptDistracted,
+  AudibleAlert.promptRepeat,
+}
+
+class QuietDriveManager:
+  def __init__(self):
+    self.params = Params()
+    self.enabled: bool = bool(self.params.get_bool("QuietDrive"))
```

Comment:
`get_bool` already returns as a bool.
```suggestion
    self.enabled: bool = self.params.get_bool("QuietDrive")
```